scene/resources/audio_stream_wav.h

```diff
@@ -157,6 +157,7 @@ class AudioStreamWAV : public AudioStream {
 	Vector<uint8_t> get_data() const;
 
 	Error save_to_wav(const String &p_path);
+	Error save_to_wav_buffer(Vector<uint8_t> wav);
```

Comment:
```suggestion
	Error save_to_wav_buffer(Vector<uint8_t> &r_wav);
```
This won't update the output, `Vector` is copy-on-write, need to pass as a reference, did you test this code?

Reply:
No, I probably should add a test in /godot/tests